onnx/defs/math/defs.cc

```diff
@@ -2942,28 +2942,24 @@ static T get_scalar_value_from_tensor(const ONNX_NAMESPACE::TensorProto* t) {
   }
 }
 
-static const char* DFT_ver17_doc = R"DOC(Computes the discrete Fourier transform of input.)DOC";
+static const char* DFT_ver20_doc = R"DOC(Computes the discrete Fourier transform of the input.)DOC";
 
 ONNX_OPERATOR_SET_SCHEMA(
     DFT,
-    17,
+    20,
     OpSchema()
-        .SetDoc(DFT_ver17_doc)
+        .SetDoc(DFT_ver20_doc)
         .Attr(
+            // TODO(justinchuby): Double check how conjugate symmetry is specified
             "onesided",
             "If onesided is 1, only values for w in [0, 1, 2, ..., floor(n_fft/2) + 1] are returned because "
-            "the real-to-complex Fourier transform satisfies the conjugate symmetry, i.e., X[m, w] = X[m,w]=X[m,n_fft-w]*. "
+            "the real-to-complex Fourier transform satisfies the conjugate symmetry, i.e., X[m, w] = X[m, w] = X[m,n_fft-w]*. "
```

Comment:
`X[m, w] = X[m, w]` is a typo? Also we should use latex here

Reply:
Looks like a duplication typo.